docs/programme/game-design/README.md

```diff
@@ -0,0 +1,49 @@
+# Game Design
+
+At the competition the robots compete by playing a game which tests their
+capabilities and provides a spectacle for guests (and the competitors). We have
+[assessment criteria](./assessment-criteria.md) for proposed games.
+
+The game played on any given year is designed well before Kickstart, though we
+hold back the publication of any details about the game in order to both
+maintain suspense and ensure that teams are all starting from the same starting
+point.
+
+We may update the rules during the course of the competition year, the rules
+document has a public version number and a section listing the amendments made
+in order to support this.
+
+## Game Selection Process
+
+We should aim to allow any volunteer to contribute an idea for a game and for
+the process of doing so to be enjoyable for all.
+
+The process typically looks like this:
+
+1. Call for submissions, where we invite everyone to submit ideas for game ideas
+   (be they whole games or mechanics which might be interesting)
+
+2. Feedback on those submissions, where general discussions happen about the
+   ideas with reference to the [assessment criteria](./assessment-criteria.md),
+   using a mixture of [mailing lists](../../volunteering/communication-platforms.md#mailing-lists),
+   [Slack](../../volunteering/communication-platforms.md#slack) and in-person or
+   video-call meetings.
+
+   During this stage, whole games are formed out of the proposed ideas. It
+   typical for semi-formal rulebooks to now be written up, as this helps force
+   thinking through the details of the game and uncovers unknowns.
+
+3. Selection of the chosen game, by the volunteers involved in the discussions
+
+4. Refinement of the chosen game, by the volunteers involved in the discussions
+
+At the start of the process deadlines for each stage will be set, typically
+allowing a few weeks for each stage (except for the selection, which happens at
+the end of the feedback stage).
+
+## Rules
+
+The rules themselves are maintained as a LaTeX project in a private repository
```

Comment:
Perhaps worth putting a link to the repo here? If someone does have access, it makes life a lot easier to see where things go on!

Reply:
I wondered about this. I ended up on the side of not linking to it, both to avoid confusion (see https://github.com/srobo/tasks/issues/195) and protect the repo. Happy to add the link if you feel strongly.

Reply:
Given https://github.com/srobo/tasks/issues/195, and potential exclusion which could be felt around not seeing a private repo, and everything around org membership, I think this is fine :+1: